### PR TITLE
Confirm the client wants to provide no email [ASANA-1204376738614787]

### PR DESCRIPTION
**Pull request by alaroia-cfa**

### CODE REVIEW

File: src/main/resources/templates/pebt/confirmBlankEmail.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+                    </div>
+                    <div class="form-card__footer">
+                        <a
+                                th:href="'/flow/pebt/contactInfo'"
+                                th:text="#{confirm-blank-email.add-email}"
+                                class="button button--primary"></a>
+                        <a
+                                th:href="'/flow/pebt/confirmBlankEmail/navigation'"
+                                th:text="#{confirm-blank-email.continue-without}"
+                                class="button button--other"
```
Comment (tdooner) [inline src/main/resources/templates/pebt/confirmBlankEmail.html:28]: Shame we don't have better fragments for these - but I think I see why you weren't able to use `continueButton` here.

File: src/main/resources/templates/pebt/confirmBlankEmail.html [hunk 2]
```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html th:lang="${#locale.language}">
+<head th:replace="~{fragments/head :: head(title=#{confirm-blank-email.title})}"></head>
+<body>
+<div class="page-wrapper">
+    <div th:replace="~{fragments/toolbar :: toolbar}"></div>
+    <section class="slab">
+        <div class="grid">
+            <div th:replace="~{fragments/goBack :: goBackLink}"></div>
+            <th:block th:if="${true}">
```
Comment (tdooner) [inline src/main/resources/templates/pebt/confirmBlankEmail.html:10]: `th:if="${true}"`?

File: src/main/resources/templates/pebt/confirmBlankEmail.html [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+                <main
+
+                        id="content" role="main" class="form-card spacing-above-35">
```
Comment (tdooner) [inline src/main/resources/templates/pebt/confirmBlankEmail.html:13]: weird spacing/indenting throughout? you may want to check on that.

File: src/main/resources/templates/pebt/confirmBlankEmail.html (hunk 1 above)
Comment (alaroia-cfa) [reply src/main/resources/templates/pebt/confirmBlankEmail.html:28]: BTW do you about this `/navigation` feature of form-flow-library? It's neat. It's a server-side HTTP endpoint that redirects to wherever `next` is, dynamically computed based on the flow. It's what continue button uses, which how I figured that out.

File: src/main/resources/templates/pebt/confirmBlankEmail.html (hunk 3 above)
Comment (alaroia-cfa) [reply src/main/resources/templates/pebt/confirmBlankEmail.html:13]: Yeah! I'm using IntelliJ Cmd-Opt-L to reformat files. I like to rely on an autoformatter. I'll see if there are configuration settings I can tweak to make that work nicely for this project; if not, I can think about not using it.

Comment (alaroia-cfa) [reply src/main/resources/templates/pebt/confirmBlankEmail.html:13]: image

This seems better.

**Pull request merged**